resources/patches/ruby_219/cve-2017-17405.patch

```diff
@@ -0,0 +1,59 @@
+From 0207c68ea39b74fc99e445231c1ac08ad5406720 Mon Sep 17 00:00:00 2001
+From: usa <usa@b2dd03c8-39d4-4d8f-98ff-823fe69b080e>
+Date: Thu, 14 Dec 2017 13:53:48 +0000
+Subject: [PATCH 1/2] merge revision(s) 61242: [Backport #14185]
+
+	Fix a command injection vulnerability in Net::FTP.
+
+git-svn-id: svn+ssh://ci.ruby-lang.org/ruby/branches/ruby_2_2@61246 b2dd03c8-39d4-4d8f-98ff-823fe69b080e
+---
+ ChangeLog                |   4 +
+ lib/net/ftp.rb           |  10 +-
+ test/net/ftp/test_ftp.rb | 234 +++++++++++++++++++++++++++++++++++++++++++++++
+ version.h                |   2 +-
+ 4 files changed, 244 insertions(+), 6 deletions(-)
```

Comment:
I kept only the `lib/net/ftp.rb` changes - `version.h` didn't apply cleanly - is that ok?